Packs/RecordedFuture/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Recorded Future v2
+**Breaking Change** the reputation commands **ip**, **domain**, **hash**, and **url** were changed to
+ return multiple entries (entry per indicator) instead of a single entry.
+ - Updated docker image from `3.8.6.13358` to `3.8.6.14516`.
```

Comment:
Add bullet.
```suggestion
- **Breaking Change** the reputation commands **ip**, **domain**, **hash**, and **url** were changed to
 return multiple entries (entry per indicator) instead of a single entry.
- Updated docker image from `3.8.6.13358` to `3.8.6.14516`.
```